app/src/main/java/fr/neamar/kiss/preference/ColorPreference.java

```diff
@@ -128,6 +142,23 @@ protected void onBindDialogView(View view) {
                 this.getPersistedString(String.format("#%08X", this.selectedColor))
         );
 
+        // This will set the correct typeface for the extra items
+        if(ColorPreference.this.selectedColor == COLOR_DARK_TRANSPARENT) {
+            Button button1 = view.findViewById(R.id.colorTransparentDark);
+            button1.setTypeface(null, Typeface.BOLD);
+            button1.setTextColor(Color.BLACK);
+        }
+        if(ColorPreference.this.selectedColor == COLOR_LIGHT_TRANSPARENT) {
+            Button button2 = view.findViewById(R.id.colorTransparentWhite);
+            button2.setTypeface(null, Typeface.BOLD);
+            button2.setTextColor(Color.BLACK);
```

Comment:
do you need to set this to black in all situations? If so, we should do this from the XML file